src/les5/Worker.java

```diff
@@ -0,0 +1,29 @@
+package les5;
+
+public class Worker {
+    private String fio = new String();
+    private String position = new String();
+    private   String email = new String();
+    private String telephone = new String();
```

Comment:
```suggestion
    private String fio = "";
    private String position = "";
    private   String email = ""
    private String telephone = "";
```
Если явно не задаем значения, то по дефолту будет null, что тоже допустимо